coral-component-icon/src/scripts/Icon.js

```diff
@@ -325,16 +325,13 @@ class Icon extends BaseComponent(HTMLElement) {
   _updateAltText(value) {
     const isImage = this.contains(this._elements.image);
     
-    let altText;
+    let altText = '';
     if (typeof value === 'string') {
       altText = value;
     }
     else if (isImage) {
       altText = '';
     }
-    else {
-      altText = this.icon.replace(SPLIT_CAMELCASE_REGEX, '$1 $2');
```

Comment:
Are you sure about removing this ?

Reply:
It creates more problems than it solves, by adding meaningless non-localized text strings like "chevron right" to elements with both icons and labels. If we were to localize all icon names from https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/coral-ui/coralui3/Coral.Icon.html#availableIcons to meaningful strings, we'd get closer to the original intent, but the current implementation masks the accessibility failures for an icon button without an accessibility name, with another accessibility failure, because the icon name provided will not necessarily be meaningful in context.

Reply:
> If we were to localize all icon names
I think this should be done maybe not for all but at least for the icon names that are meaningless like "chevron right".